pinc/faq.inc

```diff
@@ -204,14 +206,11 @@ function get_external_faq($faq, $langcode)
     return null;
 }
 
-function maybe_redirect_to_external_faq($langcode = null)
+/** @return never */
+function redirect_to_external_faq(): void
 {
-    $faq = basename($_SERVER['PHP_SELF']);
```

Comment:
This seemed convoluted that we rely on the script name when there is only one caller.

Hard-coding the value of `$faq` makes the logic clearer IMHO, in particular that we always will have a URL.